static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -0,0 +1,338 @@
+// @flow
+import { ABTest } from 'common/modules/experiments/ab-types';
+import reportError from 'lib/report-error';
+import config from 'lib/config';
+import { local } from 'lib/storage';
+import noop from 'lodash/utilities/noop';
+import abUtils from 'common/modules/experiments/utils';
+import segmentUtil from 'common/modules/experiments/segment-util';
+import testCanRunChecks from 'common/modules/experiments/test-can-run-checks';
+import acquisitionTestSelector
+    from 'common/modules/experiments/acquisition-test-selector';
+import OpinionEmailVariants
+    from 'common/modules/experiments/tests/opinion-email-variants';
+import MembershipEngagementBannerTests
+    from 'common/modules/experiments/tests/membership-engagement-banner-tests';
+import PaidContentVsOutbrain2
+    from 'common/modules/experiments/tests/paid-content-vs-outbrain';
+import { tailorSurvey } from 'common/modules/experiments/tests/tailor-survey';
+import TheLongReadEmailVariants
+    from 'common/modules/experiments/tests/the-long-read-email-variants';
+import FashionStatementEmailVariants
+    from 'common/modules/experiments/tests/fashion-statement-email-variants';
+import BookmarksEmailVariants2
+    from 'common/modules/experiments/tests/bookmarks-email-variants-2';
+import FilmTodayEmailVariants
+    from 'common/modules/experiments/tests/film-today-email-variants';
+import SleevenotesNewEmailVariant
+    from 'common/modules/experiments/tests/sleeve-notes-new-email-variant';
+import SleevenotesLegacyEmailVariant
+    from 'common/modules/experiments/tests/sleeve-notes-legacy-email-variant';
+import IncreaseInlineAdsRedux
+    from 'common/modules/experiments/tests/increase-inline-ads';
+import ophan from 'ophan/ng';
+import PaidCommenting from 'common/modules/experiments/tests/paid-commenting';
+import BundleDigitalSubPriceTest1
+    from 'common/modules/experiments/tests/bundle-digital-sub-price-test-1';
+
+let TESTS = [
+    new OpinionEmailVariants(),
+    new PaidContentVsOutbrain2(),
+    acquisitionTestSelector.getTest(),
+    tailorSurvey,
+    TheLongReadEmailVariants,
+    FashionStatementEmailVariants,
+    BookmarksEmailVariants2,
+    FilmTodayEmailVariants,
+    SleevenotesNewEmailVariant,
+    SleevenotesLegacyEmailVariant,
+    new IncreaseInlineAdsRedux(),
+    new PaidCommenting(),
+    new BundleDigitalSubPriceTest1(),
+]
+    .concat(MembershipEngagementBannerTests)
+    .filter(t => t !== undefined && t !== null);
```

Comment:
I would love to see this `TESTS` array created in a separate module. It would make this module easier to test and reduce the noise in the above `import` statements